src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_COUNT = Color.values().length;
```

Comment:
Using static fields for constants is a good practice. However, be cautious with static fields that are not constants, like `random` in line 7. It's better to avoid static methods and fields unless they are constants or there is a specific reason to use them.